go/private/rules/cgo.bzl

```diff
@@ -325,13 +345,24 @@ def setup_cgo_library(name, srcs, cdeps, copts, clinkopts):
       "external/" + REPOSITORY_NAME[1:] if len(REPOSITORY_NAME) > 1 else "",
       PACKAGE_NAME)
   copts = copts + ["-I", base_dir]
+  cxxopts = cxxopts + ["-I", base_dir]
+  cppopts = cppopts + ["-I", base_dir]
+
+  if objcopts != None:
```

Comment:
Should probably test `objc` here instead of `objcopts`.

Reply:
good catch !